[{"exception": "java.util.concurrent.ExecutionException", "inner": [{"provider": {"package": "junit.framework", "class": "TestSuite", "method": "run", "filename": "TestSuite", "line": "225"}, "caller": {"package": "org.eclipse.jdt.internal.junit.runner.junit3", "class": "JUnit3TestReference", "method": "run", "filename": "JUnit3TestReference", "line": "130"}}, {"provider": {"package": "org.eclipse.jdt.internal.junit.runner.junit3", "class": "JUnit3TestReference", "method": "run", "filename": "JUnit3TestReference", "line": "130"}, "caller": {"package": "org.eclipse.jdt.internal.junit.runner", "class": "TestExecution", "method": "run", "filename": "TestExecution", "line": "38"}}], "outer_pre": [{"provider": {"package": "org.eclipse.emf.internal.cdo.transaction", "class": "CDOTransactionImpl", "method": "commit", "filename": "CDOTransactionImpl", "line": "635"}, "caller": {"package": "org.eclipse.emf.cdo.tests", "class": "XATransactionTest", "method": "testNotUsingXATransaction_Exception", "filename": "XATransactionTest", "line": "227"}}, {"provider": {"package": "org.eclipse.emf.cdo.tests", "class": "XATransactionTest", "method": "testNotUsingXATransaction_Exception", "filename": "XATransactionTest", "line": "227"}, "caller": {"package": "junit.framework", "class": "TestCase", "method": "runTest", "filename": "TestCase", "line": "164"}}, {"provider": {"package": "junit.framework", "class": "TestCase", "method": "runBare", "filename": "TestCase", "line": "130"}, "caller": {"package": "org.eclipse.net4j.tests", "class": "AbstractOMTest", "method": "runBare", "filename": "AbstractOMTest", "line": "109"}}, {"provider": {"package": "org.eclipse.net4j.tests", "class": "AbstractOMTest", "method": "runBare", "filename": "AbstractOMTest", "line": "109"}, "caller": {"package": "junit.framework", "class": "TestResult$1", "method": "protect", "filename": "TestResult", "line": "106"}}, {"provider": {"package": "junit.framework", "class": "TestCase", "method": "run", "filename": "TestCase", "line": "120"}, "caller": {"package": "org.eclipse.net4j.tests", "class": "AbstractOMTest", "method": "run", "filename": "AbstractOMTest", "line": "131"}}, {"provider": {"package": "org.eclipse.net4j.tests", "class": "AbstractOMTest", "method": "run", "filename": "AbstractOMTest", "line": "131"}, "caller": {"package": "junit.framework", "class": "TestSuite", "method": "runTest", "filename": "TestSuite", "line": "230"}}, {"provider": {"package": "junit.framework", "class": "TestSuite", "method": "runTest", "filename": "TestSuite", "line": "230"}, "caller": {"package": "org.eclipse.emf.cdo.tests.config.impl", "class": "ConfigTestSuite$TestWrapper", "method": "runTest", "filename": "ConfigTestSuite", "line": "126"}}, {"provider": {"package": "org.eclipse.emf.cdo.tests.config.impl", "class": "ConfigTestSuite$TestWrapper", "method": "runTest", "filename": "ConfigTestSuite", "line": "126"}, "caller": {"package": "junit.framework", "class": "TestSuite", "method": "run", "filename": "TestSuite", "line": "225"}}, {"provider": {"package": "junit.framework", "class": "TestSuite", "method": "run", "filename": "TestSuite", "line": "225"}, "caller": {"package": "org.eclipse.jdt.internal.junit.runner.junit3", "class": "JUnit3TestReference", "method": "run", "filename": "JUnit3TestReference", "line": "130"}}], "outer_suf": [], "all": []}, {"exception": "java.util.concurrent.ExecutionException", "inner": [], "outer_pre": [], "outer_suf": [], "all": []}, {"exception": "org.eclipse.net4j.util.transaction.TransactionException", "inner": [], "outer_pre": [], "outer_suf": [], "all": []}]